packages/contentful/src/crn/queries/analytics.queries.ts

```diff
@@ -157,3 +157,72 @@ export const FETCH_TEAM_PRODUCTIVITY = gql`
     }
   }
 `;
+
+export const FETCH_USER_COPRODUCTION = gql`
+  query FetchUserCoproduction($limit: Int, $skip: Int) {
+    usersCollection(order: firstName_ASC, limit: $limit, skip: $skip) {
+      total
+      items {
+        sys {
+          id
+        }
+        firstName
+        lastName
+        nickname
+        alumniSinceDate
+        labsCollection(limit: 3) {
+          items {
+            sys {
+              id
+            }
+          }
+        }
+        teamsCollection(limit: 3) {
+          items {
+            team {
+              sys {
+                id
+              }
+              displayName
+              inactiveSince
+            }
+            role
+            inactiveSinceDate
+          }
+        }
+        linkedFrom {
+          researchOutputsCollection(limit: 20) {
```

Comment:
I know there's another query where we fetch at most 20 research outputs, but I think we might need to make this number higher in both queries. I was taking a look at prod, it seems there are users with more than 100 ROs, these ones for example:

https://hub.asap.science/network/users/ca867aa8-d40a-4a79-9c84-90cf8a64f013/outputs

https://hub.asap.science/network/users/e8cace05-f2cf-4e46-9de7-a7c9a537b627/outputs

Reply:
might be worth doing that work in a separate ticket for all analytics at once? This way we can focus on that problem separately and make sure we do it in a robust way.

Reply:
yeah we can't leave it that way but this is behind a FF anyway so why don't you create a ticket for it and lets add it to that same epic - we will make sure that this feature doesn't get released until its addressed